x-pack/plugins/canvas/public/state/selectors/__tests__/workpad.js

```diff
@@ -168,12 +167,6 @@ describe('workpad selectors', () => {
     });
   });
 
-  describe('getSelectedElementId', () => {
```

Comment:
CR: This definitely should not be removed, but updated for the new behavior. A test should be added for when a group is selected too.

Reply:
It's the same situation here, it's no longer exported. You're right in that I should have added a pertinent test - also for `getSelectedElement` - when groups were added. Should I add it now? It's a quick thing but it'll be a short-lived test as we should eliminate this singleton-yielding function that `getSelectedElement` is.

Reply:
> I missed that only the other selector uses it now

So this is actually OK.